src/components/player/PlayerBar.tsx

```diff
@@ -377,6 +376,7 @@ export const PlayerBar = React.memo(
                         {playlistTotalQueue && playlistTotalQueue.length > 0 ? (
                           <SongTable
                             songs={playlistTotalQueue}
+                            menuId={QUEUE_MENU_ID}
```

Comment:
This isn't supported, because it will only remove from the queue and not the playlist

Reply:
I see. I’ll fix it and fix more problem about the player.